hibernate-reactive-core/src/main/java/org/hibernate/reactive/loader/hql/impl/ReactiveQueryLoader.java

```diff
@@ -114,12 +114,20 @@ public Parameters parameters() {
 		final boolean cacheable = factory.getSessionFactoryOptions().isQueryCacheEnabled()
 				&& queryParameters.isCacheable();
 
-		String processedSQL = parameters().process( getSQLString() );
+		// Hibernate Reactive process filters just before running the query.
+		// Our parameters processor is not smart enough to work on the same query twice,
+		// so we wait to process a query if filters are enabled. It's going to be process
+		// just before the execution
+		boolean hasFilters = session.getLoadQueryInfluencers().hasEnabledFilters();
```

Comment:
@DavideD this looks perfectly fine to me

except you know how I feel about unnecessary local variables 😡😁

Reply:
I swear I'm not doing on purpose : -D

It's just that the next line reads so much better with a shorter variable

Reply:
Haha I'm one of those crazy people who thinks "more readable" ≈ "less tokens".

Reply:
I suppose it could have been a method.... well at some point I want to change this part anyway to make it easier to track where we need this stuff.

Reply:
Yeah FTR I'm also always happy to move the tokens somewhere else. 

Extract Method >> Extract Variable.

Reply:
Alright, gimme a sec